.github/workflows/test.yml

```diff
@@ -78,14 +78,8 @@ jobs:
               nextcloud-versions: ['master']
               db: ['sqlite', 'mysql', 'pgsql']
               include:
-                - php-versions: 7.4
-                  nextcloud-versions: stable22
-                  db: 'mysql'
```

Comment:
W still need to test against 7.4 because it is listed in our phpinfo.xml.

My advice: Just replace all stable* with stable25 and keep the php versions as is.

@ChristophWurst Please have a look at the workflows. What do you think?

Reply:
Agreed that we should keep coverage of all PHP versions and databases.